src/keldysh_dlr.jl

```diff
@@ -120,12 +122,23 @@ norbitals(G::DLRImaginaryTimeGF) = G.mat.norb
 # Arithmetic operations
 #
 
-function Base.:*(G::T, α::Number) where T <: DLRImaginaryTimeGF
-    return T(G.grid, α * G.mat, G.ξ)
+function Base.:+(A::T, B::T) where T <: DLRImaginaryTimeGF
+    @assert A.grid == B.grid
+    return T(A.grid, A.mat + B.mat, A.ξ)
+end
+
+function Base.:-(A::T, B::T) where T <: DLRImaginaryTimeGF
+    @assert A.grid == B.grid
+    return T(A.grid, A.mat - B.mat, A.ξ)
 end
 
-function Base.:*(α::Number, G::T) where T <: DLRImaginaryTimeGF
-    return G * α
+Base.:-(A::T) where T <: DLRImaginaryTimeGF = T(A.grid, -A.mat, A.ξ)
+
+Base.:*(G::T, α::Number) where T <: DLRImaginaryTimeGF = T(G.grid, α * G.mat, G.ξ)
+Base.:*(α::Number, G::T) where T <: DLRImaginaryTimeGF = G * α
+
+function Base.:isapprox(A::T, B::T) where T <: DLRImaginaryTimeGF
```

Comment:
Since you have added this method, I suggest you make its behavior more in line with that from `Base`.
Namely, it should throw `DimensionMismatch` if the two grids do not agree.
```
help?> DimensionMismatch
search: DimensionMismatch

  DimensionMismatch([msg])

  The objects called do not have matching dimensionality. Optional argument msg is a descriptive error string.
```

Reply:
Good suggestion! It now raises `DimensionMismatch` if the DLR grids differ and `DomainError` if their statistics (`ξ`) differ.